developer-docs-site/docs/concepts/wallet-adapter-concept.md

```diff
@@ -0,0 +1,75 @@
+- Emits events on different actions and much more
+
+Dapps should not *know* this package as dapps interact with the provider, which in turn interacts with the core package; some Types are exposed from the core package for the dapp to use.
+
+Wallets should implement their own plugin class that extends the basic plugin class (properties + events) interface that lives in the core package.
+
+:::tip
+If a wallet supports functions that are not part of the basic plugin interface, a pull request should be made to the core package to include this function so it can support it. You can take a look at the `signTransaction` on the wallet core package for guidance.
+:::
+
+### Adapter React provider
+
+The light [Adapter React package](https://github.com/aptos-labs/aptos-wallet-adapter/tree/main/packages/wallet-adapter-react) is for dapps to import and use. The package contains a `Provider` and a `Context` to implement and use within your app.
+
+Follow the [Wallet Adapter For Dapp Builders](../guides/wallet-adapter-for-dapp.md) guide on how to use the provider package on your dapp.
+
+### Adapter Template plugin
+
+Wallets looking to integrate with the adapter should implement their own wallet plugin, to ease the process we provide you with a pre-made class that implements the basic functionality needed (according to the wallet standard).
+
+The [Wallet Adapter Plugin Template repo](https://github.com/aptos-labs/wallet-adapter-plugin-template) holds a pre-made class, a test file, and some config files to help you build and publish the plugin as an NPM package.
+
+Follow the [Wallet Adapter For Wallet Builders](../guides/wallet-adapter-for-wallets.md) on how to use the template to implement and publish your wallet plugin.
+
+### Why we use this structure?
+
+When we started building a wallet adapter, we wanted a structure that can be easy enough for wallets to integrate with and for dapps to use and implement.
+
+We did research on the existing adapters, got some feedback from our community, and designed this structure to help wallets and dapps in development and when going to production. This structure offers the following benefits:
+
+* Modularity (separation of concerns) - separating the adapter into three components can help having more freedom in design, implementation, deployment and usage.
+* Wallets create and own their plugin implementation (instead of having all in the same monorepo):
+   - Reduces the packages bundle size used by dapps.
+   - Lets them be self-service and support themselves without too much friction.
+   - Prevents build failures in case of any bugs/bad implementation/wrong config files/etc.
+* Simplicity - keeps the Provider package very light and small as the major logic is implemented in the core package.
+* Flexibility - for wallets in creating and implementing custom functions.
+
+<center>
+<ThemedImage
+alt="Wallet Adapter Concept"
+sources={{
+    light: useBaseUrl('/img/docs/10-adapter-chart-light.svg'),
```

Comment:
Is there significance in the `10-` prefix of images? Can we remove it? I know Raj used numbers when more than one image exists on a page.

Reply:
not really, just following a convention I thought we have, fixed it

Reply:
Perfect. Thank you!